tests/__tests__/vueI18n.js

```diff
@@ -0,0 +1,29 @@
+import 'jest-dom/extend-expect'
+import { cleanup, render, fireEvent } from '@testing-library/vue'
+import Vuei18n from 'vue-i18n'
+import VueI18n from './components/VueI18n'
+
+afterEach(cleanup)
+
+const ja = {
+  Hello: 'こんにちは'
+}
+
+test('can render en and ja text in header', async () => {
+  const { queryByTestId, getByTestId } = render(VueI18n, {}, vue => {
+    vue.use(Vuei18n)
+    const i18n = new Vuei18n({
+      locale: 'en',
+      messages: {
+        ja
+      }
+    })
+    return { i18n }
+  })
+
+  expect(queryByTestId('section-header')).toHaveTextContent('Hello')
```

Comment:
I'd suggest using `getByText` instead of `*ByTestId`), since it is the suggested way to go (`data-testid` should be considered a escape hatch for situations where no other query can yield the desired result).

So, for example:

```suggestion
  getByText('Hello')
```

or even

```suggestion
  expect(getByText('Hello')).toBeInTheDocument()
```

(both are the same).

Same thing in other queries.